dag/main.yml

```diff
@@ -350,58 +350,50 @@ steps:
   data://grapher/happiness/2023-03-20/happiness:
   - data://garden/happiness/2023-03-20/happiness
 
-# LGBTI Policy Index (Velasco, 2020)
```

Comment:
These steps shouldn't be deleted. This happens because when you were working on your changes, there have been updates to `master` branch. You'll have to _rebase_ your branch so that your work is continuation of our work in master. Rebasing is something you do very often, typically several times per PR. You can try rebasing from VSCode (see screenshot below) against `origin/master`. You'll then resolve some conflicts and hopefully get a clean rebased version that won't be modifying anything else other than your work.

<img width="678" alt="image" src="https://github.com/owid/etl/assets/1550888/3c0a888d-ef38-4fc1-abd4-5010dcd14766">

One more tip - make a backup of your branch in case you mess something up (it's gonna happen one day...). You can do it with
```
git checkout -b backup-oil_spills
```
if you mess something up, you can reset your work with
```
git checkout oil_spills
git reset --hard backup-oil_spills
```
of course you can also do `git rebase --abort` if you feel that rebasing is not going well.